packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -93,7 +107,7 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     assert.equal(foundId, newFoundId);
   }
 
-  ['@htmlbars passing literal id is ignored and results in a default elementId'](assert) {
```

Comment:
Not sure. I just noticed that it seemed to be testing the `id` feature, so I tried running the test in glimmer and it worked.